crits/domains/api.py

```diff
@@ -64,7 +64,7 @@ def obj_create(self, bundle, **kwargs):
         ip_method = bundle.data.get('ip_method', None)
         ip_reference = bundle.data.get('ip_reference', None)
         # Also add indicators
-        add_indicators = bundle.data.get('add_indicators', None)
+        add_indicator = bundle.data.get('add_indicator', None)
```

Comment:
Hesitant to change the underlying API, but I can easily be knocked off the fence if other people want to see this change. I know the API is heavily used and this could cause problems for people who have automated systems up and running.